apps/admin/frontend/src/index.tsx

```diff
@@ -42,17 +43,18 @@ ReactDom.render(
         <ApiClientContext.Provider value={apiClient}>
           <QueryClientProvider client={queryClient}>
             <App />
+            {isFeatureFlagEnabled(
+              BooleanEnvironmentVariableName.ENABLE_REACT_QUERY_DEVTOOLS
+            ) && (
+              <div className="no-print">
+                <ReactQueryDevtools initialIsOpen={false} position="top-left" />
+              </div>
+            )}
```

Comment:
Is there a reason we moved the React Query Devtools here? I'm noticing in my local environment that I'm seeing React Query Devtools even though I don't have the env. var turned on, but that could be an issue in my environment.

Reply:
I think it doesn't work if it's outside the `QueryClientProvider`